nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -837,14 +837,11 @@ void Costmap2DROS::getCostsCallback(
 
   for (const auto & pose : request->poses) {
     geometry_msgs::msg::PoseStamped pose_transformed;
-    transformPoseToGlobalFrame(pose, pose_transformed);
-    bool in_bounds = costmap->worldToMap(
-      pose_transformed.pose.position.x,
-      pose_transformed.pose.position.y, mx, my);
-
-    if (!in_bounds) {
-      response->costs.push_back(NO_INFORMATION);
-      continue;
+    if (!transformPoseToGlobalFrame(pose, pose_transformed)) {
+      response->costs.clear();
```

Comment:
For the service level `success`? Yeah, that would make sense. 

Can't we just set these costs to `UNKNOWN`? That would essentially be the behavior before and if we can't transform it, it does seem like an "UNKNOWN" case. I could also see it being a total failure mode since not being able to TF transform one goal probably indicates some problems elsewhere.